toolkit/resources/imageconfigs/additionalfiles/iso_initrd/root/runliveinstaller

```diff
@@ -96,6 +96,13 @@ if [[ ! -f "$UNATTENDED_CONFIG_FILE" ]]; then
     # Ensure the userspace speakup connector is up
     systemctl enable espeakup
     systemctl start espeakup
+else # if an unattended iso
+    # if boot type is 'efi'
+    if [[ $(jq ".SystemConfigs[0].BootType" $UNATTENDED_CONFIG_FILE | tr -d '"') == "efi" ]]; then
```

Comment:
I also got annoyed with the double quotes but I learned that `jq` has got a flag for that.

```suggestion
    if [[ $(jq -r ".SystemConfigs[0].BootType" $UNATTENDED_CONFIG_FILE) == "efi" ]]; then
```